agentops/client.py

```diff
@@ -374,8 +378,12 @@ def end_session(
             end_state (str): The final state of the session. Options: Success, Fail, or Indeterminate.
             end_state_reason (str, optional): The reason for ending the session.
             video (str, optional): The video screen recording of the session
+            is_auto_end (bool, optional): is this an automatic use of end_session and should be skipped with skip_auto_end_session
         """
 
+        if is_auto_end and self.config.skip_auto_end_session:
```

Comment:
So in crew you're calling end_session(is_auto_end=True) ?

Reply:
correct